packages/svelte-form/tests/simple.svelte

```diff
@@ -0,0 +1,113 @@
+<script module lang="ts">
+  interface Employee {
+    firstName: string
+    lastName: string
+  }
+
+  export const sampleData: Employee = {
+    firstName: 'Christian',
+    lastName: '',
+  }
+</script>
+
+<script lang="ts">
+  import { createForm } from '@tanstack/svelte-form'
```

Comment:
There's a type error coming from here, I think it should be a relative import from `src` instead